### ISSUE TITLE
Licencing Issues

### ISSUE THREAD

**Issue by niteshgarg** [slavavdovichenko/MediaLibDemos3x#39]
Hi There,

We would like to use this lib for longer period however we are getting licencing errors and when we are trying to contact your team using the contact us form we are not able to do so and further to that we are also not able to download the lib from the web site and when we download and use from github we get licensing errors. 

Kindly guide us asap.

Thanks,
Nitesh

**Comment by markpiller** [comment]
I checked all the inquiries we received and we responded to all of them. Please contact us directly at sales@themidnightcoders.com

**Comment by niteshgarg** [comment]
Hi Mark,Thanks for your email. The major issue that we are facing right now is the issue with the MediaLib download links on your website also the contact us page doesn't work. Kindly get it checked so that we can download and check the trial version. Thanks,Nitesh                                                                                                                                                                                                                                                                                                                                        Sent from my BlackBerry 10 smartphone.                                                                                                                                                                                                                From: Mark PillerSent: Monday, 7 September 2015 22:13To: slavavdovichenko/MediaLibDemos3xReply To: slavavdovichenko/MediaLibDemos3xCc: niteshgargSubject: Re: [MediaLibDemos3x] Licencing Issues (#39)I checked all the inquiries we received and we responded to all of them. Please contact us directly at sales@themidnightcoders.com

—Reply to this email directly or view it on GitHub.

**Comment by niteshgarg** [comment]
Hi Mark,Thanks for your help and were able to download the trial now. Kindly let us know the details regarding the license. We need to implement a health care app that streams videos.Thanks,Nitesh                                                                                                                                                                                                                                                                                                                                        Sent from my BlackBerry 10 smartphone.                                                                                                                                                                                                                From: Mark PillerSent: Monday, 7 September 2015 22:13To: slavavdovichenko/MediaLibDemos3xReply To: slavavdovichenko/MediaLibDemos3xCc: niteshgargSubject: Re: [MediaLibDemos3x] Licencing Issues (#39)I checked all the inquiries we received and we responded to all of them. Please contact us directly at sales@themidnightcoders.com

—Reply to this email directly or view it on GitHub.

**Comment by markpiller** [comment]
I sent the licensing information to you email address.

**Comment by ilitvinenko90** [comment]
Hello I'm try to use MediaLibDemos3x (MediaLibiOS), but I got:
"connectFailedEvent: -7 = fault code = -7 < A license is needed to use a library. Contact Midnight Coders Sales to purchase the license >"

Could you please give me more info about licensing and how can I get it?

Here is my email: evgen.litvinenko.dev@gmail.com

Thank you in advance

**Comment by ilitvinenko90** [comment]
Also here is Skype id: onegin166
I think it will be much faster to communicate via Skype. 
Please help me it is very important for me right now, I got client how wan't to finish it asap.

When I try to add it in at my project, in crashed or Annoying sound from device dynamic.

**Comment by markpiller** [comment]
Evgen,

We have thousands of users and providing direct support via skype is not very efficient from our perspective.

Regards,
Mark

**Comment by ilitvinenko90** [comment]
Could I have any support regarding my issues, 
"When I try to add it in at my project, in crashed or Annoying sound from device dynamic."?

Thank you,
Evgen

**Comment by markpiller** [comment]
Have you tried any of the samples included with it? Do they work for you? If they do, then configure your project exactly the same way as the samples.

**Comment by ilitvinenko90** [comment]
When I run only example it works well, but when I integrate it in my project I got annoying sound. 
I used custom mode, because I need to send iPad screen as output source.

**Comment by markpiller** [comment]
You can contact sales@themidnightcoders.com to arrange a support license.

**Comment by ilitvinenko90** [comment]
So main issue in license, right?

**Comment by markpiller** [comment]
No, the main issue is that you're doing something wrong. You will need to purchase a support package so we can help you solve your problem.

**Comment by ilitvinenko90** [comment]
For now I have 2 issue:
Annoying sound and sometimes streaming are stopped due license or this "[flv @ 0x14585ac00] Encoder did not produce proper pts, making some up."

How much would it cost, to fix them, or help me to understand how can I solve this issues?

**Comment by markpiller** [comment]
To get a pricing quote contact sales@themidnightcoders.com
